### PR TITLE
Removed round brackets for plural strings.

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

Signed-off-by: rakekniven <mark.ziegler@rakekniven.de>

### CODE REVIEW

**Review by er-vin — COMMENTED** [review]
This looks wrong to me, it's actually intended for Qt handling of plural forms:
https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals

File: src/libsync/syncengine.cpp [hunk 1]
```diff
@@ -349,7 +349,7 @@ void OCC::SyncEngine::slotItemDiscovered(const OCC::SyncFileItemPtr &item)
                 const auto result = _syncOptions._vfs->convertToPlaceholder(filePath, *item);
                 if (!result) {
                     item->_instruction = CSYNC_INSTRUCTION_ERROR;
-                    item->_errorString = tr("Could not update file : %1").arg(result.error());
+                    item->_errorString = tr("Could not update file: %1").arg(result.error());
```
Comment (er-vin) [inline src/libsync/syncengine.cpp:352]: That one is correct indeed. I'd totally merge a PR with just that commit. ;-)

Comment (rakekniven) [reply src/libsync/syncengine.cpp:352]: See https://github.com/nextcloud/desktop/pull/2860

**Comment by rakekniven** [comment]
> This looks wrong to me, it's actually intended for Qt handling of plural forms:
> https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals

So you are right. Closing this.

**Pull request closed without merging**